.github/workflows/markdownlint.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v4
       with:
-        node-version: 20.x
+        node-version: 'lts/*'
```

Comment:
This is the LTS syntax as recommended by NVM, which is the syntax used by this action. https://github.com/nvm-sh/nvm?tab=readme-ov-file#long-term-support